### DISCUSSION TITLE
Launch Camera: null is not an object (evaluating '_reactNative.NativeModules.ImagePickerManager.launchImageLibrary')

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [react-native-image-picker/react-native-image-picker#2121] [Q&A]
Hi 

After upgrading `react-native-image-picker` from `4.8.1` to `5.3.1`, I'm having problems with a project having the following dependencies:

```
  "react-native": "0.64.2",
  "react-native-image-picker": "5.3.1"
```
On iOS16 the `launchCamera` method works fine, however, if I run the application on Android API 30 with the following `build.gradle` file:

```
ext {
        buildToolsVersion = "29.0.3"
        minSdkVersion = 21
        compileSdkVersion = 29
        targetSdkVersion = 29
        ndkVersion = "20.1.5948944"
}
``` 

When I call the `launchCamera` method I receive the following error:

```
[TypeError: null is not an object (evaluating '_reactNative.NativeModules.ImagePickerManager.launchCamera')]
```

How can I solve this issue?

Thanks

**Comment by MirandaMiranda181840** [comment]
Having the exact same issue here!
